terraform/aws/variables.tf

```diff
@@ -153,3 +153,8 @@ variable "logs_node_disk_size" {
   description = "Disk size of log nodes"
   default     = 100
 }
+
+variable "logs_node_instance_type" {
```

Comment:
It seems it highly depends on size of the network, but elastc recommends 2xlarge as minimal setup. Probably we should set xlarge at least?

Reply:
I think this is something we will have to learn over time. For smaller networks with ~12 nodes (theoretically this will be the primary use of the tool, right?) then I think t3.medium would be ideal. For testnet, we obviously need something larger, depending on what we are logging.

Reply:
Yeah. It would be nice to have in future automatic adjustments based on number of nodes. So you set only how many masternodes you want and the tool generate approprate configs for terraform and ansible.